runtime-modules/content/src/lib.rs

```diff
@@ -2781,6 +2790,87 @@ impl<T: Trait> Module<T> {
             .cloned()
             .collect::<BTreeSet<_>>()
     }
+
+    fn ensure_sufficient_balance_for_channel_transfer(
+        owner: &ChannelOwner<T::MemberId, T::CuratorGroupId>,
+        transfer_cost: BalanceOf<T>,
+    ) -> DispatchResult {
+        let balance = match owner {
+            ChannelOwner::Member(member_id) => {
+                let controller_account_id =
+                    T::MemberAuthenticator::controller_account_id(*member_id)?;
+
+                Balances::<T>::usable_balance(&controller_account_id)
```

Comment:
I assume this means that any funds that have been received as part of invitation would not count? as this would be a fund leak. If so, please add a comment to that effect reminding the reader.

Reply:
Correct. Added a comment.